airbyte-integrations/connectors/source-tiktok-marketing/integration_tests/spec.json

```diff
@@ -5,43 +5,13 @@
     "title": "TikTok Marketing Source Spec",
     "type": "object",
     "properties": {
-      "credentials": {
```

Comment:
this will break existing configs, so we shouldn't rename it

Reply:
All OLD auth_types:
    prod_access_token
    sandbox_access_token
    oauth2.0

#1. If I just remove the odd auth method (oauth2.0) then old "OAuth2.0" related config fails validation.

So in order to support all possible old configs I renamed 'credentials' to 'credentials_all'.
It allows successful validation for all possible configs.


#2. But if we know that all old "OAuth2.0" config can be updated to new format (and use prod_access_token instead) then 'credentials' can be left unchanged.

@sherifnada   what do you think about #2, I don't know how we manage "OAuth2.0" in airbyte cloud.

Reply:
@midavadim my apologies I didn't see the application logic change below. I see now how this was maintaining backwards compatibility. 

IMO the simpler thing to do here would be to remove the `prod_access_token` -- my guess is most people are using oauth or sandbox. if it fails, they'll have to fix it, but renaming the whole object means that everyone using tiktok will have to re-set their configuration in the UI. I think until we have migrations that's a better outcome than having an old branch in the code that lives forever. So I would support removing prod_access_token